<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#131239;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font:13pt Arial Rounded MT Bold;
color: white;</string>
   </property>
   <property name="text">
    <string>Are you sure you want to reset account?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="f_popup_yes">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>120</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Arial Rounded MT Bold&quot;;
padding:5px;
background-color:green;
color:white;

</string>
   </property>
   <property name="text">
    <string>YES</string>
   </property>
  </widget>
  <widget class="QPushButton" name="f_popup_no">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>120</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Arial Rounded MT Bold&quot;;
padding:5px;
background-color:green;
color:white;

</string>
   </property>
   <property name="text">
    <string>NO</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
